### DISCUSSION TITLE
increaseViewportBy vs overscan

### DISCUSSION THREAD

**Discussion by kevinjmo** [petyosi/react-virtuoso#1035] [Q&A]
Would someone mind explaining clearly the difference between these two? I can’t quite seem to get it down based on the information in the docs. When would one use `increaseViewportBy` versus `overscan`?

> In a nutshell, increasing the `overscan` causes less frequent re-renders, but makes each re-render more expensive (because more items will get replaced).

There’s documentation that talks about the performance of `overscan` but nothing about `increaseViewportBy`.

**Comment by gusapps** [comment]
+1 and these props also don't seem to work with `useWindowScroll={true}`
